_posts/2019-05-20-foreman-community-newsletter---may-2019.md

```diff
@@ -0,0 +1,156 @@
+it a spin and let us know how you get on - and do [report issues][issues] as
+you find them! We also had a test week kindly organized by Lukas[lzap] in the
+simillar fashion as done in the previous release. We couldn't do this without
+the testing we get from the community, so thanks very much to you all!
+
+[1_22_rc1]: https://community.theforeman.org/t/foreman-1-22-0-rc1-is-ready-for-testing/13766
+[issues]: https://projects.theforeman.org/projects/foreman/issues
+[release_notes]: https://theforeman.org/manuals/1.22/index.html#Releasenotesfor1.22
+
+As ever, many thanks to the contributors to these releases - coders, RC testers,
+documentation writers, and bug/feature reporters. Foreman wouldn't be what it
+is without you!
+
+### Conference roundup
+
+As ever, Foreman was present at Red Hat Summit this year,
+and as usual the Foreman/Pulp booth looked incredibly busy every time the expo
+area open. The talks by Tomer and Ewoud were well received. As Tomer
+mentioned in the community demo, a lot of questions were asked around Ansible
+integration, working of foreman with Ansible Tower, Remote execution and also
+acknowledged the improvements to content view publishing, better UX for repos
+and audit. Sounds like good news to us! All the work that we have been putting
+in is being noticed :)
+
+Not to forget, we had the very awesome yellow Foreman hats distributed at the Foreman
+booth. Thanks to Tomer and Ori for handling the design and shipment of the goodies.
+
+### Community Survey analysis is now out
+
+Thanks to everyone who filled out this year's community survey - we have now
+the analysis of the report that was prepared by Greg. We had 116 replies, which is a
+little down on last year, but still more than enough data to work with. As promised, Greg has
+written us a blog. It should be up and running any time :)
+
+### [Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+All Foreman events are available via [the forum
+calendar](https://community.theforeman.org/calendar) - hit the Subscribe button
+(which will give you an ICS link). You can also use the "Add To Calendar" button
+on specific events if you don't wish to import the whole thing.
+
+* [OpenExpo Europe](https://community.theforeman.org/t/openexpo-europe/13467)
+
+Tomer will be giving a talk about “Hybrid cloud management with Foreman” at the conference.
+He will be talking about some of the main features of Foreman, and how you can leverage it to simplify and unify your datacenter management. All the best Tomer :)
+
+* [Puppet Contributor Summit](https://community.theforeman.org/t/puppet-contributor-summit/13985)
+
+The puppet contributor summit is on its way this year in the month of June. It majorly
+focusses on the discussions  around the present and the future of Puppet projects. Btw, if
+you plan on attending the conference, then dont forget to use the discount code `theForeman50` to get 50% off :slight_smile:
+
```

Comment:
maybe add foreman 10th birthday ? (e.g. not sure when we will have the next newsletter?)

Reply:
Sorry for being late on the newsletters, but we will have them every month, in fact, I intend to bring in a few new features in the newsletter form the next month :)

As mentioned, I think it is a good idea to publicize the event from this month itself :) thanks @ohadlevy  :+1: